database/migrations/2018_04_02_150700__labels_display_model.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class LabelsDisplayCompanyName extends Migration
```

Comment:
FYI, this looks like some copypasta from a previous migration. This classname was used before, so it throws an error. I fixed it in https://github.com/snipe/snipe-it/commit/b4542d4d42ba6457c2c8c53b5cb7f28ae3c4259c